### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by aboydnw**
The PR template seems a little outdated, so updating with some checklists of frequent items that come up during reviews.

### CODE REVIEW

**Review by acblackford — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
-- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
-- [Other](?expand=1&template=default.md)
\ No newline at end of file
+Adding a Data Story:
+- [ ] Any data referenced in the story is already published to VEDA STAC and referenced in a separate dataset overview page (prior to merging or as part of the same PR)
+- [ ] All metadata is completed in the mdx file
+- [ ] Any images are compressed, ideally <500 KB
+- [ ] Published Date is as close to the actual release date as possible (work with veda-ui team to determine potential release date when story is close to ready)
+- [ ] At least one code owner has reviewed the PR
```
Comment (dzole0311) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Should we add a third one for when we only pull-in the latest `veda-ui` updates?

**Review by dzole0311 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (aboydnw) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: @dzole0311 I like that idea! I'm not very familiar with that process though. Would you or @sandrahoang686 @hanbyul-here be able to suggest a checklist?

Comment (hanbyul-here) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: oki I took a stab and pushed the change. (I also edited the doc about updating veda-ui) Let me know how it looks!

**Comment by hanbyul-here** [comment]
but I have a question - did you list out the possible options on purpose? Currently, the PR asks you what kind of change you are making and takes you to the right template - Your approach can save a click so I don't really mind the change, but I just wanted to confirm if this was intentional (If so, we need to delete the files in the folder https://github.com/US-GHG-Center/veda-config-ghg/tree/develop/.github/PULL_REQUEST_TEMPLATE )

**Review by dzole0311 — APPROVED** [review]

**Comment by aboydnw** [comment]
@hanbyul-here the PR template you linked to is in veda-config-ghg. I don't think we have the same template(s) for veda-config. Maybe it's just about pulling over those PR templates? If they work better, I'm open to doing that too.

**Comment by hanbyul-here** [comment]
@aboydnw Urg sorry, I meant these files  : https://github.com/NASA-IMPACT/veda-config/tree/develop/.github/PULL_REQUEST_TEMPLATE 
I really do not have a good idea if just removing the unrelated parts is more convenient for users or clicking to the specific one is better. But we just need to be coherent (if we are making a user to delete unrealted parts, remove the files in `PULL_REQUEST_TEMPLATE`, if we want a user to follow the previous workflow, split the current contents into each template and retrieve the previous links)

**Pull request merged**